### DISCUSSION TITLE
Make Acode compatible with VS Code Extensions.

### DISCUSSION THREAD

**Discussion by Mesum-Hussain** [Acode-Foundation/Acode#991] [Ideas]
Acode is going on the right path and has become a very powerful app ever since I discovered it 2 years ago.

But the only thing that's preventing me from making it my daily driver code editor is the sheer amount of VS Code Extensions I am used to and love on my PC. If Acode becomes compatible with VS Code Extensions and all those Extensions become available on Acode Extension Store then there's literally nothing stopping Acode from becoming a worthy alternative to VS Code on Android.

I would happily switch all my React Native projects from my PC to my Android Tablet for working on the go.

**Comment by bajrangCoder** [comment]
It's kind off impossible to make Vscode extension with the Acode because both have different architecture and platform support but It's possible to rewrite those extensions for acode
